modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder/form_builder_dd_support.js

```diff
@@ -374,13 +374,15 @@ AUI.add(
 				var instance = this;
 
 				if (activeLayout.get('rows').length == 1 && instance._verifyEmptyForm(activeLayout.get('rows')[0]) && !A.one('#lfr-initial-col-message')) {
+					var colNode = activeLayout.get('rows')[0].get('cols')[0].get('node');
```

Comment:
@clovisdasilvaneto Could you please rename it to `columnNode` and remove the chaining `activeLayout.get('rows')[0].get('cols')[0].get('node')` ? Thx